crowdloans/rewards/polkadot.json

```diff
@@ -336,16 +336,23 @@
         },
         {
           "label": "Kintsugi Legends Bonus",
-          "formula": "${amount * 3.463 * 0.025}",
+          "content": "+2.5% INTR",
           "tooltip": [
-            "2.5% if you contributed to kintsugi crowdloan with same wallet"
+            "2.5% if you contributed to kintsugi crowdloan with same wallet till 12/23/21"
           ]
         },
         {
           "label": "Quiz Challenge Bonus",
-          "formula": "${amount * 3.463 * 0.025}",
+          "content": "+2.5% INTR",
           "tooltip": [
-            "2.5% for solving the Interlay quiz, see below in Reward Rules for the link to their website"
+            "2.5% for solving the Interlay quiz, see below in Reward Rules for the link to their website till 12/23/21"
+          ]
+        },
+        {
+          "label": "Test Mint Challenge Bonus",
+          "content": "+2.5% INTR",
+          "tooltip": [
+            "2.5% for minting interBTC on testnet till 12/23/21"
```

Comment:
Update to: 2.5% for minting interBTC on testnet before December 23, 2021, December 23, 2021, 18:00 UTC.